src/components/core/testing/private/a11y-tree-snapshot.ts

```diff
@@ -45,9 +45,10 @@ export function testA11yTreeSnapshot(
       await a11yTreeEqualSnapshot();
     });
 
-    testIf(isSafari() && !exclude.safari, 'Safari', async () => {
-      await a11yTreeEqualSnapshot();
-    });
+    // TODO: Try if tests on CI are no longer flaky
+    // testIf(isSafari() && !exclude.safari, 'Safari', async () => {
+    //   await a11yTreeEqualSnapshot();
+    // });
```

Comment:
Just a nitpick, we can also remove 'safari' from the `exclude` param at line 34

Reply:
Asked @kyubisation and we would like to keep to option for the moment